tests-fuzz/src/ir.rs

```diff
@@ -355,6 +355,20 @@ pub fn droppable_columns(columns: &[Column]) -> Vec<&Column> {
         .collect::<Vec<_>>()
 }
 
+/// Returns columns that can use the alter table modify command
+pub fn modifiable_columns(columns: &[Column]) -> Vec<&Column> {
+    columns
+        .iter()
+        .filter(|column| {
+            !column.options.iter().any(|option| {
+                option == &ColumnOption::PrimaryKey
+                    || option == &ColumnOption::TimeIndex
+                    || option == &ColumnOption::NotNull
+            })
```

Comment:
The `Nullable` and `TimeIndex` columns should also be modifiable. cc @evenyag

Reply:
It should not be supported at present.
https://github.com/GreptimeTeam/greptimedb/blob/4e5dd1ebb04d910bdb88611a8f824529701606b8/src/table/src/metadata.rs#L487-L494

Reply:
I see